app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,36 @@
+module MarkdownHelper
+  class HTMLwithCoderay < Redcarpet::Render::HTML
+    def block_code(code, language)
+      lang = language.blank? ? "md" : language
+      CodeRay.scan(code, lang).div
+    end
+  end
+
+  def markdown(text)
+    unless @markdown
+      options = {
+        filter_html: true,
+        hard_wrap: true
+      }
+      extentions = {
```

Comment:
__[Misspell]__

"extentions" is a misspelling of "extensions"


<!-- Comment by [SideCI](https://sideci.com) -->